conf/application.conf

```diff
@@ -153,6 +155,11 @@ microservice {
       port = 8500
     }
 
+    bank-holidays {
```

Comment:
is this available also as back-end microservice available in service manager?

Reply:
Not that I'm aware of, it's just a hosted file. I just did it like this so we can override it in each env as if it was

Reply:
A few other projects use the bank-holidays.json and it seems all of them just use the gov.uk url https://github.com/search?p=2&q=org%3Ahmrc+bank-holidays&type=Code

Reply:
e.g. https://github.com/hmrc/vat-registration-frontend/blob/821005b1022dd2412d76cf69189e1f89b2a9a1f2/app/connectors/BankHolidaysConnector.scala